### PR TITLE
Ignore dev dependencies dependabot updates

### PR DESCRIPTION
**Pull request by bidoubiwa**
ll

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -14,3 +14,4 @@ updates:
   ignore:
       - dependency-name: "eslint*"
       - dependency-name: "*prettier*"
+      - dependency-name: "*jest*"
```
Comment (bidoubiwa) [inline .github/dependabot.yml:17]:
```suggestion
      - dependency-name: "*jest*"
      - dependency-name: "*cypress*"
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -14,3 +14,5 @@ updates:
...
+      - dependency-name: "*cypress*"
```
Comment (bidoubiwa) [inline .github/dependabot.yml:18]:
```suggestion
      - dependency-name: "*cypress*"
      - dependency-name: "* concurrently*"
```

**Review by brunoocasali — APPROVED** [review]
Let's make dependabot great again! 🎉

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**